assets/src/js/header.js

```diff
@@ -1,87 +1,87 @@
-export const setupHeader = function($) {
+export const setupHeader = function() {
   'use strict';
 
-  $(document).on('click', [
+  document.querySelector(document).addEventListener('click', [
     '.navbar-dropdown-toggle',
     '.country-dropdown-toggle',
     '.navbar-search-toggle',
   ].join(), function toggleNavDropdown(evt) {
     evt.preventDefault();
     evt.stopPropagation();
 
-    const $button = $(this);
-    const target = $button.data('bs-target');
+    const button = document.querySelector(this);
+    const target = button.data('bs-target');
     if (!target) {
       throw new Error('Missing `data-bs-target` attribute: specify the container to be toggled');
     }
-    const toggleClass = $button.data('bs-toggle');
+    const toggleClass = button.data('bs-toggle');
     if (!toggleClass) {
       throw new Error('Missing `data-bs-toggle` attribute: specify the class to toggle');
     }
 
     // Toggle visibility of the target specified via data-bs-target.
-    $(target).toggleClass(toggleClass);
-    $(this).toggleClass(toggleClass);
+    document.querySelector(target).classList.toggle(toggleClass);
+    document.querySelector(this).classList.toggle(toggleClass);
 
     // Toggle aria-expanded attribute.
-    $button.attr('aria-expanded', function(i, attr) {
+    button.attr('aria-expanded', function(i, attr) {
       return attr === 'false' ? 'true' : 'false';
     });
 
     // Toggle data-ga-action attribute used in GTM tracking.
-    $('.country-dropdown-toggle').attr( 'data-ga-action', $('.country-dropdown-toggle').attr('aria-expanded') === 'false' ? 'Open Country Selector' : 'Close Country Selector' );
-    $('.navbar-search-toggle').attr( 'data-ga-action', $('.navbar-search-toggle').attr('aria-expanded') === 'false' ? 'Open Search' : 'Close Search' );
+    document.querySelector('.country-dropdown-toggle').attr( 'data-ga-action', document.querySelector('.country-dropdown-toggle').attr('aria-expanded') === 'false' ? 'Open Country Selector' : 'Close Country Selector' );
+    document.querySelector('.navbar-search-toggle').attr( 'data-ga-action', document.querySelector('.navbar-search-toggle').attr('aria-expanded') === 'false' ? 'Open Search' : 'Close Search' );
   });
 
   // Close all menus when clicking somewhere else
-  $(document).on('click', function closeInactiveMenus(evt) {
+  document.querySelector(document).addEventListener('click', function closeInactiveMenus(evt) {
     const clickedElement = evt.target;
-    $('button[aria-expanded="true"]').each(function(i, button) {
-      const $button = $(button);
-      const buttonTarget = $($button.data('bs-target')).get( 0 );
-      if (buttonTarget && ! $.contains(buttonTarget, clickedElement)) {
+    document.querySelector('button[aria-expanded="true"]').each(function(i, button) {
+      const button = document.querySelector(button);
```

Comment:
This raises an error, since `button` is defined multiple times in that loop.

Reply:
Should i use <code>let</code> for <code>button</code> AND <code>buttonTarget</code>?

Reply:
Not sure if that would work. You have already declared a variable with the name `button` a few lines above (line 12)

Reply:
Hey @himanshu007-creator! Did you had the time to check it out?

Reply:
hi @comzeradd, I saw it, had exams. I tried to figure it out, but didn't work

Reply:
I'll de-assign the relevant tickets, in case someone else wants to give it a try. But I'll leave this PR open for now, if you find the time to have another look.